src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Task {
+
```

Comment:
I think this blank line can be removed? (So that it follows the formatting of the rest of your code)